README.md

```diff
@@ -12,6 +12,7 @@
 - JS,
 - Arafat's feature
 - Testing this branch feature
+- Adding content to branch
```

Comment:
We cannot accept this, pls review the code and revert.

Reply:
Done